### DISCUSSION TITLE
strelka-backend status error problem

### DISCUSSION THREAD

**Discussion by ldceven** [Security-Onion-Solutions/securityonion#2152] [Unsupported Versions] [answered]
Hello, my sensor node has been running for a period of time and found that the strelka-backend service will become ERROR. Is this normal?
[root@dc-sec-so-sensor-01 ~]# sudo so-status

Checking Docker status

    Docker ------------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------------------ [OK]

Checking container statuses

    so-filebeat ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- --------------------------- [OK]
    so-nginx ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------------------ [OK]
    so-sensoroni ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- -------------------------- [OK]
    so-steno ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------------------ [OK]
    so-strelka-backend --------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------- [ERROR]
    so-strelka-coordinator --------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------ [OK]
    so-strelka-filestream --------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------- [OK]
    so-strelka-frontend --------------------------------------------- -------------------------------------------------- -------------------------------------------------- --------------------- [OK]
    so-strelka-gatekeeper --------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------- [OK]
    so-strelka-manager --------------------------------------------- -------------------------------------------------- -------------------------------------------------- ---------------------- [OK]
    so-suricata ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- --------------------------- [OK]
    so-telegraf ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- --------------------------- [OK]
    so-wazuh ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------------------ [OK]
    so-zeek ----------------------------------------------- -------------------------------------------------- -------------------------------------------------- ------------------------------- [OK]

The docker log is as follows:
[root@dc-sec-so-sensor-01 stenographer]# docker logs -f --follow 1ccf28acddbe
2020-12-01 07:03:31-[INFO] root [strelka-backend.main]: using backend configuration /etc/strelka/backend.yaml
2020-12-01 07:03:31-[DEBUG] root [strelka-backend.main]: verified coordinator is up
2020-12-01 07:03:31-[INFO] root [strelka-backend.work]: starting up
2020-12-01 07:18:31-[INFO] root [strelka-backend.work]: shutdown after scanning 32 file(s) and 900.0909237861633 second(s)

**Comment by 0nnyx** [comment, marked as answer]
Search for similar problems : https://github.com/Security-Onion-Solutions/securityonion/discussions/2127
